### PR TITLE
Try bumping cirq_core

### PR DESCRIPTION
**Pull request by lillian542**
Bump cirq to version 1.3.0 in the `requirements.txt` files, and bump all the other things that need to be bumped to make that work.

### CODE REVIEW

File: doc/requirements.txt [hunk 1]
```diff
@@ -8,14 +8,14 @@ Babel==2.9.1
 cachetools==4.2.4
 certifi==2021.5.30
 charset-normalizer==2.0.12
-cirq==0.14.1
-cirq-core==0.14.1
-cirq-pasqal==0.14.1
+cirq==1.0.0
+cirq-core==1.0.0
+cirq-pasqal==1.0.0
```
Comment (lillian542) [inline doc/requirements.txt:13]:
```suggestion
cirq==1.3.0
cirq-core==1.3.0
cirq-pasqal==1.3.0
```

Comment (lillian542) [inline doc/requirements.txt:13]:
```suggestion
cirq==1.3.0
cirq-core==1.3.0
cirq-pasqal==1.3.0
```

File: requirements.txt [hunk 2]
```diff
@@ -4,11 +4,12 @@ autoray==0.3.1
 cachetools==5.0.0
 certifi==2021.10.8
 charset-normalizer==2.0.12
-cirq-core==0.14.1
-cirq-pasqal==0.14.1
+cirq==1.0.0
+cirq-core==1.0.0
+cirq-pasqal==1.0.0
```
Comment (lillian542) [inline requirements.txt:9]:
```suggestion
cirq==1.3.0
cirq-core==1.3.0
cirq-pasqal==1.3.0
```

File: doc/requirements.txt [hunk 3]
```diff
@@ -8,14 +8,14 @@ Babel==2.9.1
...
-cirq-pasqal==0.14.1
+cirq==1.3.0
+cirq-core==1.3.0
+cirq-pasqal==1.3.0
 cycler==0.10.0
 decorator==5.1.1
 deprecation==2.1.0
 docutils==0.16.0
-duet==0.2.5
+duet==0.2.7
```
Comment (lillian542) [inline doc/requirements.txt:18]:
```suggestion
duet==0.2.8
```

File: requirements.txt [hunk 4]
```diff
@@ -4,11 +4,12 @@ autoray==0.3.1
...
 charset-normalizer==2.0.12
-cirq-core==0.14.1
-cirq-pasqal==0.14.1
+cirq==1.3.0
+cirq-core==1.3.0
+cirq-pasqal==1.3.0
 cycler==0.10.0
 deprecation==2.1.0
-duet==0.2.5
+duet==0.2.7
```
Comment (lillian542) [inline requirements.txt:12]:
```suggestion
duet==0.2.8
```

File: doc/requirements.txt [hunk 5]
```diff
@@ -8,18 +8,18 @@ Babel==2.9.1
...
 decorator==5.1.1
 deprecation==2.1.0
 docutils==0.16.0
-duet==0.2.5
+duet==0.2.8
 fonttools==4.32.0
 future==0.18.3
 google-api-core==1.31.5
-google-auth==1.35.0
+google-auth==1.36.0
```
Comment (lillian542) [inline doc/requirements.txt:22]:
```suggestion
google-auth==2.0.0
```

File: doc/requirements.txt [hunk 6]
```diff
@@ -8,18 +8,18 @@ Babel==2.9.1
...
+cirq-pasqal==1.3.0
 cycler==0.10.0
 decorator==5.1.1
 deprecation==2.1.0
 docutils==0.16.0
-duet==0.2.5
+duet==0.2.8
 fonttools==4.32.0
 future==0.18.3
 google-api-core==1.31.5
```
Comment (lillian542) [inline doc/requirements.txt:21]:
```suggestion
google-api-core==2.0.0
```

File: doc/requirements.txt [hunk 7]
```diff
@@ -8,18 +8,18 @@ Babel==2.9.1
...
 deprecation==2.1.0
 docutils==0.16.0
-duet==0.2.5
+duet==0.2.8
 fonttools==4.32.0
 future==0.18.3
-google-api-core==1.31.5
-google-auth==1.35.0
+google-api-core==2.0.0
+google-auth==2.0.0
```
Comment (lillian542) [inline doc/requirements.txt:22]:
```suggestion
google-auth==2.25.2
```

File: doc/requirements.txt [hunk 8]
```diff
@@ -8,18 +8,18 @@ Babel==2.9.1
 cachetools==4.2.4
```
Comment (lillian542) [inline doc/requirements.txt:8]:
```suggestion
cachetools==5.3
```

File: requirements.txt [hunk 9]
```diff
@@ -4,11 +4,12 @@ autoray==0.3.1
 cachetools==5.0.0
```
Comment (lillian542) [inline requirements.txt:4]:
```suggestion
cachetools==5.3
```

**Review by mudit2812 — APPROVED** [review]

**Comment by timmysilv** [comment]
not sure entirely how to handle this, but I tried to add `cirq-core==1.3.0` to `qml` dependencies explicitly, and poetry gave me an error:
```
qml on  master [$!?] is 📦 v0.0.0 via 🐍 3.9.13 (.venv) took 17s 
❯ poetry add -G executable-dependencies cirq-core==1.3.0

Updating dependencies
Resolving dependencies... (1.0s)

Because cirq (1.0.0) depends on cirq-core (1.0.0)
 and no versions of cirq match >1.0.0,<1.1.0 || >1.1.0,<1.2.0 || >1.2.0,<1.3.0, cirq (>=1.0.0,<1.1.0 || >1.1.0,<1.2.0 || >1.2.0,<1.3.0) requires cirq-core (1.0.0).
And because cirq (1.2.0) depends on cirq-core (1.2.0), cirq (>=1.0.0,<1.1.0 || >1.1.0,<1.3.0) requires cirq-core (1.0.0 || 1.2.0).
And because cirq (1.1.0) depends on cirq-core (1.1.0)
 and mitiq (0.29.0) depends on cirq (>=1.0.0,<1.3.0), mitiq (0.29.0) requires cirq-core (1.0.0 || 1.1.0 || 1.2.0).
So, because qml depends on both mitiq (0.29.0) and cirq-core (1.3.0), version solving failed.
```

basically, we pin `mitiq==0.29.0`, and that needs `cirq-corq<1.3.0`. If we can bump `mitiq` to `0.32.0`, we'll be fine. But the _bigger_ question is, do we care? it's not an explicit requirement here anyway

**Review by albi3ro — APPROVED** [review]
Thanks for actually doing this 🎉

**Comment by lillian542** [comment]
> not sure entirely how to handle this, but I tried to add cirq-core==1.3.0 to qml dependencies explicitly, and poetry gave me an error

note: we discussed offline and decided this is fine for now, something to think about standardizing going forward

**Pull request merged**